mxnet/inference/docker/1.9/py3/Dockerfile.cpu

```diff
@@ -173,7 +194,8 @@ RUN HOME_DIR=/root \
  && chmod +x /usr/local/bin/testOSSCompliance \
  && chmod +x ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh \
  && ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh ${HOME_DIR} python \
- && rm -rf ${HOME_DIR}/oss_compliance*
+ && rm -rf ${HOME_DIR}/oss_compliance* \
+ && rm -iRf /root/.cache
```

Comment:
Do we need to add this for the ec2 image as well? I see test failures on ec2 which might get resolved by adding this.

Reply:
I pushed a fix for this.